docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -14,6 +14,7 @@ The following table lists the available UI components:
 | Control                              | Description                              | Related classes                          |
 | ------------------------------------ | ---------------------------------------- | ---------------------------------------- |
 | [Button](./nui-components/Button.md) | A button that can set action when user select it. | `Button`                |
+| [CanvasView](./vectorgraphics/CanvasView.md) | A CanvasView is a class for displaying vector primitives. | `CanvasViewView` |
```

Comment:
CanvasView is a kind of Component in NUI.
Other than that, Shape, Picture, and DrawableGroup are VectorGraphics classes that can be added to CanvasView.
The first time I put CanvasView Page on two pages: ui-components and VectorGraphics. 
Since this is a duplicate, me and my team decided to put it only on ui-components.
If there is an exact rule or guide for this, please let me know.
I can leave the CanvasView in the list of components.md page and move the CanvasView.md page to VectorGraphics.

Reply:
Hello @JSUYA 
Thanks for your reply. If you want so, you can keep CanvasView in UI Components.

However, as for now there are two cover pages linked to CanvasView, how about removing link to CanvasView from the list in vectorgraphics/Overview.md? 
As there is another referencing link marked as 1), I think we can remove 2)

![image](https://user-images.githubusercontent.com/9958374/148493333-137b51b8-da1f-44c5-bb43-f4950203fb69.png)

Reply:
Oh I missed the title of this page, "Overview".

How about "VectorGraphics" instead of "Overview"?

Reply:
> Hello @JSUYA Thanks for your reply. If you want so, you can keep CanvasView in UI Components.
> 
> However, as for now there are two cover pages linked to CanvasView, how about removing link to CanvasView from the list in vectorgraphics/Overview.md? As there is another referencing link marked as 1), I think we can remove 2)
> 
> ![image](https://user-images.githubusercontent.com/9958374/148493333-137b51b8-da1f-44c5-bb43-f4950203fb69.png)

I removed 2).

Reply:
> How about "VectorGraphics" instead of "Overview"?

I modified it. Thank you